codespell_lib/tests/test_basic.py

```diff
@@ -174,7 +172,7 @@ def test_interactivity(tmpdir, capsys):
         with FakeStdin('0\n'):  # blank input -> nothing
             assert cs.main('-w', '-i', '3', f.name) == 0
         assert cs.main(f.name) == 0
-        with open(f.name, 'r') as f_read:
```

Comment:
Not sure about this one either. What's wrong with the explicit `r'`? It echoes the 'w' below.

Reply:
`'r'` is the default, so is unnecessary:

https://docs.python.org/3/library/functions.html#open

It is conventional within the larger Python community to omit it. The goal of this PR is to update the code to follow wider community conventions.